presidio-analyzer/presidio_analyzer/predefined_recognizers/transformers_recognizer.py

```diff
@@ -0,0 +1,69 @@
+from typing import Tuple, Set
+import logging
+from presidio_analyzer.predefined_recognizers.spacy_recognizer import SpacyRecognizer
+from presidio_analyzer import RecognizerResult
+
+logger = logging.getLogger("presidio-analyzer")
+
+
+class TransformersRecognizer(SpacyRecognizer):
+    """
+    Recognize entities using the transformers package.
+
+    See https://huggingface.co/docs/transformers/main/en/index
+    Uses the transformers package
+    (https://huggingface.co/docs/transformers/main/en/installation) to align
+    transformers interface with spaCy
```

Comment:
Worth mentioning that this recognizer doesn't run transformers models, but loads the output from the NlpArtifacts